client/src/components/ratingsReviews/RatingsReviews.jsx

```diff
@@ -236,8 +233,13 @@ export default function RatingsReviews() {
     const getMetaApi = async () => {
       try {
         const res = await axios.get(
-          `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/meta/?count=50&product_id=${selectedProduct.id}`,
+          'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/meta',
           {
+            params: {
+              count: 50,
+              product_id: selectedProduct.id
+            },
+
```

Comment:
Overall this structure looks nice and a bit easier to read. Well done!

Reply:
Thanks!